config/environments/test.rb

```diff
@@ -8,7 +8,7 @@
   config.cache_classes = true
 
   # Log error messages when you accidentally call methods on nil.
-  config.whiny_nils = true
+  # config.whiny_nils = true
```

Comment:
You are right, I thought I saw it being set in config/environments/development.rb  but I was wrong sorry :)